metadata.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "puppet-letsencrypt",
-  "version": "5.0.1-rc0",
+  "version": "6.0.1-rc0",
```

Comment:
I don't think this is a major version.

Reply:
yes. please don't update the version. we do this during the next release.